### PR TITLE
Enabled configurable compression + decompression of values with...

### PR DESCRIPTION
**Pull request by jcacciatore**
… DynoJedisClient for String and Hash commands. Set and List commands are not supported since this would break sort operations.

### CODE REVIEW

File: dyno-contrib/src/main/java/com/netflix/dyno/contrib/ArchaiusConnectionPoolConfiguration.java [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/*******************************************************************************
+ * Copyright 2011 Netflix
```
Comment (ipapapa) [inline dyno-contrib/src/main/java/com/netflix/dyno/contrib/ArchaiusConnectionPoolConfiguration.java]: change date to 2015

File: dyno-core/src/main/java/com/netflix/dyno/connectionpool/CompressionOperation.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright 2011 Netflix
```
Comment (ipapapa) [inline dyno-core/src/main/java/com/netflix/dyno/connectionpool/CompressionOperation.java]: update date

File: dyno-core/src/main/java/com/netflix/dyno/connectionpool/ConnectionPoolConfiguration.java [hunk 3]
```diff
@@ -154,4 +162,18 @@
...
+     * <p>
+     * The value for this configuration setting is specific in terms of <strong>kilobytes</strong>
+     *
+     * @return The final value to be used as a threshold. Note that a negative value disables this feature.
+     */
+    int getValueCompressionThreshold();
+
+    CompressionStrategy getCompressionStrategy();
+
 }
```
Comment (ipapapa) [inline dyno-core/src/main/java/com/netflix/dyno/connectionpool/ConnectionPoolConfiguration.java]: I see "No new line at end of file"

**Comment by ipapapa** [comment]
Looks good overall. Fixing the travis-ci build issue will be nice.

**Comment by jcacciatore** [comment]
Thanks for reviewing @ipapapa. I'll certainly fix the unit test issue before proceeding.

**Pull request merged**